GitHub-Org-Management-Policy.md

```diff
@@ -0,0 +1,70 @@
+# Node.js GitHub Organization Management Policy
+
+The Node.js Foundation GitHub Organization (https://github.com/nodejs) is
+provided as a development resource by the Node.js Foundation under the direction
+of the Node.js Technical Steering Committee (TSC) and Node.js Foundation
+Community Committee (CommComm).
+
+## Node.js Admin Repository
+
+The [Node.js admin repository](https://github.com/nodejs/admin) will serve as
+the central location for managing Node.js GitHub Organization administrative
+activities. Only Node.js GitHub Organization owners, TSC members, and Community
+Committee members will have write permissions to the Node.js admin repository.
+
+## Organization Roles
+
+### Owners
+
+TSC members and the Chair of the Community Committee are the only individuals granted
+Owner permissions within the Node.js GitHub Organization.
+
+### Members
+
+GitHub users are added as members to the Node.js GitHub Organization when they
+are added to any Working Group. Organization Owners should add new members
+to the organization when requested by a Working Group.
```

Comment:
`...by a Working Group or a team.`

Also it may be worth mentioning the owner should suggest new members to follow https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md and enable 2FA prior to adding them to the organization.